corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py

```diff
@@ -94,7 +94,7 @@ def testBasicRebuild(self):
         case.xform_ids = [case.xform_ids[0], case.xform_ids[2], case.xform_ids[1]]
         case.rebuild()
         self.assertEqual(case.p1, 'p1-1') # original
-        self.assertEqual(case.p2, 'p2-1') # updated (back!)
+        self.assertEqual(case.p2, 'p2-2') # updated (back!)
```

Comment:
Oh, I was assuming it was testing that rebuilding with out of order actions but perhaps that's what the next test is doing. I'll set the dates to the same value which should put it back into the previous state.